packages/neuron-ui/src/components/Transaction/index.tsx

```diff
@@ -16,23 +16,25 @@ const inputColumns: IColumn[] = [
     key: 'lockHash',
     name: 'Lock Hash',
     minWidth: 100,
-    maxWidth: 525,
-  },
-  {
-    key: 'outPointBlockHash',
-    name: 'OutPoint BlockHash',
-    minWidth: 150,
-    onRender: (item: any) => <span>{item.previousOutput.blockHash || 'none'}</span>,
+    maxWidth: 200,
+    onRender: (item: any) => (
+      <span title={item.lockHash || 'none'} className="text-overflow">
+        {item.lockHash || 'none'}
+      </span>
+    ),
   },
   {
     key: 'outPointCell',
     name: 'OutPoint Cell',
```

Comment:
I think `outPoint` is enough.

Reply:
For now, I just keep it working in the desired way. We can remove it in the next version.